tests/app/dao/notification_dao/test_notification_dao.py

```diff
@@ -2094,27 +2089,25 @@ def test_dao_get_count_of_letters_to_process_for_today_without_notis_does_not_ra
 
 @freeze_time("2017-12-18 17:50")
 def test_dao_get_count_of_letters_to_process_for_date_ignores_service_not_letters_as_pdf(
```

Comment:
Would this be better?

test_dao_get_count_of_letters_to_process_for_date_only_counts_letters_as_pdf?